app/controllers/admin/api/registry/policies_controller.rb

```diff
@@ -39,6 +39,7 @@ def authorize_policies
   end
 
   def policy_params
-    params.require(:policy).permit(:name, :version, :schema)
+    policy_params = params.require(:policy).permit(:name, :version)
+    policy_params.merge(schema: params.require(:policy).require(:schema)).permit!
```

Comment:
Ok, not a problem, but anyway the model checks that it is not empty and otherwise it responds that it is invalid 😄

Reply:
Well this is because we can send either a `Hash` or a `String` for `schema`
Better require it in controller so we know that it is valid to be used by model.